Redirect the users to sonarsource-cfamily-examples for C, C++ and Obj-C (#143)
SonarSource/sonar-scanning-examples: README.md

[SonarQube-Scanner-Swift Code Coverage](swift-coverage/)

> **_NOTE:_** All examples related to C, C++ and Objective-C can be found here: [sonarsource-cfamily-examples](https://github.com/sonarsource-cfamily-examples)

Reviewer: ```suggestion
**_NOTE:_** All SonarQube Scanner examples for C, C++ and Objective-C can be found [here](https://github.com/sonarsource-cfamily-examples?q=-sq).
```

Reviewer: The scope of this repo seems to be primarily SQ scanner examples. I restricted the link to those. Users can find out by reading the README there that we cover a wider scope.
WDYT?

Author: I hesitated TBH :)
So that's fine with me!
